libraries/testing/tester.cpp

```diff
@@ -333,7 +333,7 @@ namespace eosio { namespace testing {
          vector<transaction_id_type> scheduled_trxs;
          while ((scheduled_trxs = get_scheduled_transactions()).size() > 0 ) {
             for( const auto& trx : scheduled_trxs ) {
-               auto trace = control->push_scheduled_transaction( trx, fc::time_point::maximum(), DEFAULT_BILLED_CPU_TIME_US );
+               auto trace = control->push_scheduled_transaction( trx, fc::time_point::maximum(), DEFAULT_BILLED_CPU_TIME_US, false );
```

Comment:
I think the prior behavior was wrong. It seems to me that this logic should match what producer_plugin does. Hopefully soon this can be replace by a producer library call. I've made `push_transaction` & `push_scheduled_transaction` match what producer_plugin does.

Reply:
@arhag convinced me we don't want timing behavior causing issues here as this is in the core tester.